project/src/main/java/com/google/sps/servlets/QueryServlet.java

```diff
@@ -49,16 +54,35 @@ void init(PlacesFetcher inputFetcher) {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     try {
```

Comment:
If possible, I think the try block should be more focused on the lines causing the handeled exception.

Reply:
That's a great point. This try-catch is in order to send an error to the "fetch" call, so that we'll go to the catch there if there is any error in the servlet (so that the code doesn't break). what to you think is the right scope for the "try"? I'm not sure about it.

Reply:
From the catch message you wrote I understand that it handles fetching problems, so probably surround only the fetch call (or maybe all lines 65 - 70). But really whatever you think.